hooks/hooks.py

```diff
@@ -68,6 +115,13 @@ def relation_changed():
         if render_file(n, template_data) or not host.service_running(n):
             host.service_restart(n)
 
+    # Render the file that makes the kubernetes binaries available to minions.
+    if render_file(
+            'distribution', template_data,
+            'conf.tmpl', '/etc/nginx/sites-enabled/distribution') or \
```

Comment:
Will we be adding multiple versions for this? I thought we were going to do multiple vhosts per version deployed for distribution. Perhaps i'm remembering this incorrectly.